### ISSUE TITLE
Sensor Fusion Algorithm Implementation

### ISSUE THREAD

**Issue by digitaldanny** [digitaldanny/GestureControl#64]
**Is your feature request related to a problem? Please describe.**
Having trouble getting the DMP working on the MPU6050. I can use an external sensor fusion library to get accurate euler angles from the raw accelerometer and gyroscope readings.

**Describe the solution you'd like**
- Convert raw accelerometer and gyroscope data into hand orientation values on 2 axes.
  - The algorithm will use gyroscope and accelerometer values. There will be drift on the third axis, but I do not plan on using this axis for the final demo.
  - Capture the gyro error offset while sitting still. These values will need to be subtracted from all gyro readings.
- Create a millis() implementation to capture the time difference between sensor data captures.
- How much can I compress these values before transferring over Bluetooth?

**Additional context**
- #47 - MPU6050 I2c Drivers
- The issue linked above also has references to an all-inclusive sensor fusion library. Might be able to quickly make use of this to get orientation values.

**Comment by digitaldanny** [comment]
Copying over the sensor fusion repo link - [here](https://github.com/rbv188/IMU-algorithm).

**Comment by digitaldanny** [comment]
Created a quick Arduino sketch that reads raw sensor data from the MPU6050. Using the Arduino Serial Plotter, I was able to see the data changing live as I moved the sensor. Moving or rotating the breakout board along one axis at a time allowed me to verify that valid data was being read from the sensor for both accelerometer and gyroscope sensors.

**Gyroscope Plot - Rotating 1 axis at a time**
![image](https://user-images.githubusercontent.com/40513675/113789558-5288d400-970d-11eb-947f-f89cdcbc8e5a.png)

**Accelerometer Plot - Moving 1 axis at a time**
![image](https://user-images.githubusercontent.com/40513675/113789801-de9afb80-970d-11eb-9ee2-e5065fa7ffa8.png)
